src/matcha_ml/services/global_parameters_service.py

```diff
@@ -119,8 +119,9 @@ def default_config_file_path(self) -> str:
         Returns:
             The default global configuration directory.
         """
-        home_path = os.path.expanduser("~")
-        return os.path.join(home_path, ".matcha-ml", "config.yaml")
+        config_path = os.path.join(os.path.expanduser("~"), ".config")
+
+        return os.path.join(config_path, "matcha-ml", "config.yaml")
```

Comment:
Does `config_file` function create a config folder if not present?

Reply:
Yep, in this line: https://github.com/fuzzylabs/matcha/blob/72003c2dda40b663c8ea64202a6f820e9bbdcc4f/src/matcha_ml/services/global_parameters_service.py#L65